### PR TITLE
script: unwind aura and aurabal bpt to vault

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
issue #642 

edit: withdraws all from the aura and aurabal bpt straight to the vault msig

### CODE REVIEW

**Review by petrovska-petro — DISMISSED** [review]
Clean and snapshots looks good ✅

File: scripts/issue/642/adjust_bpt_bootstraps.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ]
+    safe.take_snapshot(tokens)
+    voter.take_snapshot(tokens)
+
+    safe.balancer.unstake_all_and_withdraw_all(
+        pool=bpt_aura,
+        pool_type='non_stable',
+        destination=r.badger_wallets.treasury_voter_multisig
+    )
+    safe.aura.deposit_all_and_stake(bpt_aurabal)
```
Comment (jalbrekt85) [inline scripts/issue/642/adjust_bpt_bootstraps.py:31]: on my end, `assert self.booster.depositAll(pool_id, stake).return_value == True` check fails but deposit goes through successfully without the check

Comment (gosuto-inzasheru) [reply scripts/issue/642/adjust_bpt_bootstraps.py:31]: what is `type` or value of the `.return_value` on your side?

Comment (jalbrekt85) [reply scripts/issue/642/adjust_bpt_bootstraps.py:31]: returns `None`

**Comment by gosuto-inzasheru** [comment]
in light of #655 all (gravi)aura liquidity should be moved to the vault.

**Review by petrovska-petro — APPROVED** [review]
LGTM! ✅ 

loads vault with more graviaura & aura for digg tcl move

**Review by jalbrekt85 — APPROVED** [review]
adjusted based on #655, lgtm

**Pull request merged**